docs/reference/sql/functions/date-time.asciidoc

```diff
@@ -404,6 +404,48 @@ include-tagged::{sql-specs}/docs/docs.csv-spec[dateDiffDateTimeMinutes]
 include-tagged::{sql-specs}/docs/docs.csv-spec[dateDiffDateMinutes]
 --------------------------------------------------
 
+[[sql-functions-datetime-dateformat]]
+==== `DATE_FORMAT/FORMAT/TO_CHAR/TOCHAR`
+
+.Synopsis:
+[source, sql]
+--------------------------------------------------
+DATE_FORMAT(
+    datetime_exp/time_exp, <1>
+    string_exp) <2>
+--------------------------------------------------
+
+*Input*:
+
+<1> date/datetime/time expression
+<2> format pattern
+
+*Output*: string
+
+*Description*: Returns the date/datetime/time as a string using the format specified in the 2nd argument. The formatting
+pattern used is the one from
+https://docs.oracle.com/en/java/javase/14/docs/api/java.base/java/time/format/DateTimeFormatter.html[`java.time.format.DateTimeFormatter`].
+If any of the two arguments is `null` or the pattern is an empty string a `null` is returned.
```

Comment:
drop a : is string `null` is returned.
If the pattern is empty, null be returned - is that the behavior of the other DBs?

Reply:
MySQL and PostgreSQL return `null` for empty pattern, SQL Server returns string in the default format.

Reply:
Let's go with the PostgreSQL.